metron-interface/metron-alerts/src/app/alerts/alerts-list/auto-polling/auto-polling.service.spec.ts

```diff
@@ -486,18 +489,18 @@ describe('AutoPollingService', () => {
 
       expect(searchServiceFake.search).toHaveBeenCalledTimes(1);
 
-      tick(getIntervalInMS());
+      tick(localAutoPollingSvc.getInterval() * 1000);
```

Comment:
You may want to consider normalizing your abstraction for the tick calculations, i.e. you're using a nice function `getIntervalInMS()` in some places, but removed it here bc you're using a `localAutoPollingSvc` rather than the global `autoPollingService`. Not a blocker, but something to consider if you're inclined.